src/Route.elm

```diff
@@ -20,6 +22,12 @@ toString route =
         Index ->
             "/"
 
+        CaseStudy ->
```

Comment:
This is great as a first step. We'll want it to take a parameter and be `case-study/title`.